app.js

```diff
@@ -13,10 +13,11 @@ const dateFilter = require('nunjucks-date-filter');
 const config = require('./app/config');
 const locals = require('./app/locals');
 
-class App {
-  constructor() {
+export class App {
```

Comment:
are you not tempted to remove this class as well (and api/manifest providers)???

Reply:
probably not this one. Since we need to pass in the AuthProvider dependency to this now when we are creating the app for running or when testing. 

But yeah I think the api and manifest providers can be de-classed. I'll do them as part of a separate PR.

Reply:
I think this one can be done as well ... 😄

Reply:
tech debt